Readme.md

```diff
@@ -38,9 +38,8 @@ steps:
 
 Buildkite Compat can also be used via a HTTP API using `rackup` from the `app` folder of this repository.
 
-Note: If you are using the docker image you will have to override the entrypoint:
-```shell
-$ docker run --rm -ti -p 9292:9292 --entrypoint '' --workdir /app $IMAGE:$TAG rackup --port 9292
+```sh
+docker-compose up app
```

Comment:
I would add it as an option to run the project but wouldn't replace the raw `docker` command.